gulp-core-build-typescript/src/TSLintTask.ts

```diff
@@ -115,6 +117,10 @@ export class TSLintTask extends GulpTask<ITSLintTaskConfig> {
     const taskScope: TSLintTask = this;
 
     const activeLintRules: any = taskScope._loadLintRules(); // tslint:disable-line:no-any
+
+    // @todo This stream is not piped to returned value. Find a way to block the execute flow.
+    this._writeLintRules(activeLintRules);
```

Comment:
If you want to block behind this stream, I would advice using merge plugin to merge this and the below streams.

However, I still think using gulp to write a single file is complete overkill. fs.writeFileSync() will be 10% the # of lines.